data/companies/axel_springer/property_data.yaml

```diff
@@ -3,7 +3,8 @@
 type: publisher
 company: Axel Springer
 properties:
-  - template: news_with_print
+  - template: generic
```

Comment:
@bokelley Is this right? I saw you removed the `new_with_print` template, so I believe this is the right change but want to check

Reply:
yes